recipe/meta.yaml

```diff
@@ -31,9 +31,10 @@ requirements:
     # Depend on galaxy-util during build time to avoid packaging galaxy/__init__.py .
     - galaxy-util >={{ galaxy_version }}
   run:
+    - typing-extensions
```

Comment:
Mmmh, this and the following change from grayscull are not good, should we revert https://github.com/conda-forge/galaxy-tool-util-feedstock/pull/16 ?

Reply:
Yes, https://github.com/galaxyproject/galaxy/blob/dev/packages/tool_util/setup.cfg#L44 but this line duplicates `typing_extensions` (with an underscore) which we already have below.

In this case it is kinda harmless, since `typing-extensions` is just an empty package that requires `typing_extensions`, but I think we want to keep the `galaxy-util` pin as it was.